packages/inference/src/providers/fal-ai.ts

```diff
@@ -1,31 +1 @@
-import type { ProviderMapping } from "./types";
-
 export const FAL_AI_API_BASE_URL = "https://fal.run";
-
-type FalAiId = string;
-
-export const FAL_AI_SUPPORTED_MODEL_IDS: ProviderMapping<FalAiId> = {
```

Comment:
same comment as in https://github.com/huggingface/huggingface_hub/pull/2836/files#r1942755169, i would for now keep the mappings (but make them empty)
and still support them in code

That way we have slightly less breaking changes too, potentially